java/src/test/java/com/google/crypto/tink/subtle/AesSivTest.java

```diff
@@ -110,6 +110,11 @@ public void testEncryptDecryptWithEmptyPlaintext() throws GeneralSecurityExcepti
         byte[] rebuiltPlaintext = dead.decryptDeterministically(ciphertext, aad);
         assertEquals(AesUtil.BLOCK_SIZE, ciphertext.length);
         assertEquals(Hex.encode(plaintext), Hex.encode(rebuiltPlaintext));
+
+        byte[] nullAadCiphertext = dead.encryptDeterministically(plaintext, null);
```

Comment:
Please move this to a separate unit test.

I think we also want to test the following scenarios:
- Encrypting with non-empty plaintext and null AAD
- Using Null AAD is equivalent to using empty AAD